packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -197,9 +205,9 @@ export interface IContextualMenuItem {
   href?: string;
 
   /**
-   * A collection of submenu items
+   * A collection of submenu items, or an IContextualMenuProps object to apply for the sub menu
    */
-  items?: IContextualMenuItem[];
+  items?: IContextualMenuItem[] | IContextualMenuProps;
 
```

Comment:
Oh, I would be totally fine with renaming `items` as `subMenuProps` with the union type, except that it would break back-compat. And I feel that having both `items` and `subMenuProps` would be confusing and ill-defined.

If we're okay with a major version change, I can do the rename.